src/aznfswatchdog

```diff
@@ -221,14 +225,18 @@ reconcile_conntrack()
 
     if [ -n "$output" ]; then
         for entry in $output; do
-            # XXX Remove this log after running for few days.
+            # XXX Remove this log after running for few days.(not too noisy!)
             vecho "$entry"
 
-            matchstr=".* SYN_SENT src=.* dst=$l_ip sport=([0-9]+) dport=([0-9]+).*"
+            # pasting here for ease during review (remove later)
+            # tcp      6 114 SYN_SENT src=10.20.0.17 dst=10.161.100.101 sport=819 dport=2048 [UNREPLIED] src=20.150.35.196 dst=10.20.0.17 sport=2048 dport=819 mark=0 use=1
+            matchstr="tcp \s*[0-9]+ \s*([0-9]+).* SYN_SENT src=.* dst=$l_ip sport=([0-9]+) dport=([0-9]+).*"
```

Comment:
since regexp tries to find the longest match we have to be careful that the sport and dport that match are not in the reply direction. Also to be safe let's use \s* for space as it can be a space or tab or multiple spaces.

matchstr="tcp\s+[0-9]+\s+([0-9]+)\s+SYN_SENT\s+src=[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+\s+dst=${l_ip}\s+sport=([0-9]+)\s+dport=([0-9]+).*"